extensions/elasticsearch/7.x/wazuh-template.json

```diff
@@ -327,6 +327,7 @@
       "data.vulnerability.title",
       "data.vulnerability.assigner",
       "data.vulnerability.cve_version",
+      "data.vulnerability.alert_origin",

```

Comment:
This new alert field is to report where the alert was generated... it could take the vaules "NVD feed" or "Vendor feed".

Reply:
I decided to compose a string that could take three different values depending on the alert origins. The values are `NVD and vendor feeds`, `NVD feed` or `Vendor feed`.